lua/admin.lua

```diff
@@ -191,7 +191,7 @@ function rels_proc (kv, nokeys)
            end
          end
        end
-       if kv["name"] == "England" or kv["name"] == "Scotland" or kv["name"] == "Wales" or kv["name"] == "Northern Ireland" then
+       if kv["name"] == "England" or kv["name"] == "Alba / Scotland" or kv["name"] == "Cymru / Wales" or kv["name"] == "Northern Ireland" then
```

Comment:
> doesnt this mean that someone converting an older planet will also get screwed? i think the compatible fix would be to include both variants right? just support both for a time. its a bit of baggage to carry but its better than getting bug reports :D

@kevinkreiser Yeah I had the following, but was concerned folks would complain.  Thoughts?

```
if kv["name"] == "Alba / Scotland" then
   kv["name"] = "Scotland"
end

if kv["name"] == "Cymru / Wales" then
   kv["name"] = "Wales"
end
```

Reply:
IMO fine fix:)

Reply:
I had actually meant just leave both checks in and use whichever we find. Which means doing it at the other places where you have changes as well. Basically support both names all the way through the code

Reply:
Yep I know that's what I'm suggesting

Reply:
Do we know when those relations changed? If it was a long while ago then just ignore my comment. If it was recent then it's a pretty huge gotcha to have to know to use the bleeding edge changeset

Reply:
@kevinkreiser Scotland was 8 months ago and Wales I am not sure as it keeps timing out on me.

Reply:
ok cool then we can just ship it as is. we'll assume wales was also a while ago. sorry for the noise i just didnt want to knowingly put a gotcha in there

Reply:
@kevinkreiser np.  thanks for the feedback